### DISCUSSION TITLE
Support for multiple stores

### DISCUSSION THREAD

**Discussion by jegra** [nmaier95/craft-shopify-product-fetcher#36] [Ideas]
Would it be possible to add support for multiple stores within a single Craft installation?  Thinking about a situation where a client has different Shopify stores for different countries, and would like to serve content from a single, localized base site.

**Comment by jacobgraf** [comment]
Yes. 100% this. With this added, we would be able to fully do (natively) what we've been working on hacking together for the last 6-months. Would love to see this added as well!

**Comment by emson-admin** [comment]
This would also be wonderful over here. Multiple extension stores under shopify plus with a dream of using craft to extend and sync content.

**Comment by jacobgraf** [comment]
Definitely a big miss here for Craft to support multi-sites but not support multiple Shopify stores. We share 90% of our content between stores. We've waited for years and have had to move towards other solutions which really sucks because we absolutely love Craft and not being able to use it kills me. Argh.

**Comment by jegra** [comment]
Disappointing news.  We were under the impression that it would be completed within a year's time, which is why we'd decided to move forward with this as a solution for our client's site.  Multi-site with multi-store would be an amazing differentiator for Craft, and would make it a go-to solution for these types of site requirements.  Hope it gets another look at some point...

**Reply by jacobgraf** [reply]
We were in the same boat. We were also told it was being worked on. We were patient. Very patient. Pinged a handful of times for updates and got radio silence. I put a bad review on the Craft Plugin Store the other day expressing my frustration and now magically the feature branch is deleted and we're being told it's not coming. Argh. Maddening. Like I said, I love Craft, but this really is a miss, and a very frustrating one for people in the same boat we are.
